src/caffeine/allocation_s.f90

```diff
@@ -20,12 +20,10 @@
     integer :: me
     type(c_ptr) :: whole_block
     integer(c_ptrdiff_t) :: block_offset
-    integer(c_size_t) :: handle_size, coarray_size, total_size
+    integer(c_size_t) :: handle_size, total_size
```

Comment:
Weak request to rename the misleading variable `handle_size` to `descriptor_size`, because that's what it actually represents.